### ISSUE TITLE
Does support jira server as bug tracker?

### ISSUE THREAD

**Issue by Nov1kov** [MicrosoftDocs/appcenter-docs#672]
I used Jira as a bug tracker In Hockey App successfully via login and password.
App Center requires use jira via app token.
But Jira server doesn't support feature API tokens 
https://confluence.atlassian.com/cloud/api-tokens-938839638.html
This feature only for Jira cloud.
What can I do?
Thanks.

---
#### Document details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 99771c68-5f45-887a-e926-9ea772021f99
* Version Independent ID: 865109fc-64b8-ba77-2e10-8002886c46c7
* Content: [App Center bug tracker documentation - Visual Studio App Center](https://docs.microsoft.com/en-gb/appcenter/dashboard/bugtracker/#feedback)
* Content Source: [docs/dashboard/bugtracker/index.md](https://github.com/MicrosoftDocs/appcenter-docs/blob/live/docs/dashboard/bugtracker/index.md)
* Service: **vs-appcenter**
* GitHub Login: @amchew
* Microsoft Alias: **amchew**

**Comment by nishaatr** [comment]
I have the same question and have asked AppCenter chat. Waiting for a response. 
Looks like OAuth might work but setting up OAuth is involved so not pursuing this avenue at the moment.
Another option might be to use email notification from AppCenter into JIRA to create the issue and attachments.  Waiting on AppCenter support before we explore this option.

**Comment by samuele-churchill** [comment]
Same question here

**Comment by zayass** [comment]
Same issue

**Comment by anywherepilot** [comment]
Jira removed the support for this feature. There is a link to the deprecation documentation from Jira in the App Center documentation. If you believe that user/password login still works for the server version of Jira, please open a feature request for supporting this version of Jira in the main App Center repository and up-vote it: https://github.com/microsoft/appcenter/issues/

Feature requests which gather many votes in the main App Center repository have a good change of making it into the product soon.

**Comment by King-of-Spades** [comment]
closing this in favor of https://github.com/microsoft/appcenter/issues/1789
